app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,19 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.includes(:actors).find(params[:id])
+    @actors = @movie.actors.order(age: :asc)
+    @average_age = @actors.average(:age)
+  end
+
+  def update
+    @movie = Movie.find(params[:id])
+    
+    redirect_to @movie
+  end
+  
+  private
+  
+  def movie_params
+    params.require(:movie).permit(:actor_id)
```

Comment:
The form syntax you're using in the view is a bit old -- consider form_with url and then you won't have to use the `require(:movie)` and can just permit :actor_id